File: example_solutions/python/test4.py
def ans(f):
 P=open(f).read().split('\n');S=[];p=lambda:len(S)and S.pop();m=n=J=0;d=5
 while 64-J:
  O=P[n][m:];O=O and O[0]or" ";Z=O in'\\:$!@`_|'and p();Q=Z==0;d={60:3,62:5,94:1,118:7,95:[3,5][Q],124:[1,7][Q]}.get(J:=ord(O),d);n=(n+d//3-1)%len(P);m=(m+d%3-1)%80;S+=[Z,Z]*(58==J)+[Q]*(33==J)+[J-48]*(47<J<58)
  if O in'+-*/%`':a=p();S+=[int(eval(f"{p()}{O if J-96else'>'}{a}"))]
  if 92==J:S+=[Z,p()]
 return Z

File: example_solutions/javascript/test4.js
ans=f=>{for(P=require('fs').readFileSync(f,'utf8').split("\n"),S=[],p=x=>S.pop(),m=n=0,d=5;q=x=>S.push(~~x);m+=79+d%3,n+=29+~~(d/3)){var O=P[n%30]?.[m%80]||' ',I="<>^v_|\\:!@$+-*/%`".indexOf(O),Z=I>3&&p()
I>10?q(eval(p()+(I-16?O:'>')+Z)):I-6?I-7?I-8?+O|O=='0'&&q(+O):q(!Z):q(Z)+q(Z):q(Z,f=p())+q(f)
d=[3,5,1,7,Z?3:5,Z?1:7][I]||d
if(I==9)return Z}}